lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1590,7 +1595,7 @@ static bool ParseClangImporterArgs(ClangImporterOptions &Opts,
 
   // Forward the FrontendOptions to clang importer option so it can be
   // accessed when creating clang module compilation invocation.
-  if (FrontendOpts.EnableCaching)
+  if (RequiresCAS)
```

Comment:
This should still be `EnableCaching` as I don't think MCCAS affect anything in clangImporter.